src/templates/_common/_layout/layout.jsx

```diff
@@ -62,7 +62,7 @@ const Layout = () => {
     }
 
     return (
-        <html>
+        <html className='light'>
```

Comment:
@prince-deriv i know there is no dark theme for SmartTrader at the moment. But it doesn't feel right to hard-code 'light' right here. Can we please make it dynamic, or use some config for controlling whether it's light or dark?

Reply:
Sure @maryia-deriv  I will check a way to store it somewhere